testsuite/testsuite.py

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/env python
+
+"""
+Usage::
+
+    testsuite.py [OPTIONS]
+
+Run the adareducer testsuite. This requires e3-testsuite.
+"""
+
+import os
+
+import e3.testsuite
+
+from drivers.shell_script import ShellScriptDriver
+
+
+class AFLControlTestsuite(e3.testsuite.Testsuite):
```

Comment:
Should probably be AdareduceControlTestsuite? :)

Reply:
Copypasta. Fixed!